README.md

```diff
@@ -14,7 +14,21 @@ Additional description about the project and its features.
 
 ## Live Demo
 
-- [Live Demo Link](https://mstanley-me.github.io/toDoList/)
+- [Live Demo Link](https://mstanley-me.github.io/TODO/)
+
+
+## Getting Started
+
+- To get a local copy up and running follow these simple example steps.
+
+# Install
+
+- After cloning run ` npm install`
+
+# Usage
+
+run `python -m http:server` or `npm install -g live-server` after run `live-server`
```

Comment:
- [ ] I would suggest removing the live link section for now, if you haven't deployed it yet. It's getting `404 not found`.

- [ ] Also you wrote `after cloning` but you haven't mentioned how to clone. Please write all the steps so that the user wouldn't confused.

``` 
git clone https://github.com/stanleySimeon/TODO.git

cd TODO

and then the rest of the steps
```

Reply:
This is a github problem!

Reply:
Hello @stanleySimeon, please make sure you implement the changes requested by the previous code reviewer. You just need to update your `readme file` with the above commands, before the instructions on how to run your app.

Reply:
It's okay now you can take a look
Thak you so much